### ISSUE TITLE
Where find PyHSCam.pyd

### ISSUE THREAD

**Issue by Julesghes** [ion201/PyHSCam#3]
Hello, 
Thank you for your work, I want to control Photron SA3.
I installed python3.5 , VS2015, succeeded bootstrap ,
But to try the exemple, i need PyHSCam module
Where is it? in pip or github

ImportError: No module named 'PyHSCam'

**Comment by ion201** [comment]
Unfortunately I can't offer much assistance for this project as I do not use windows here, nevermind having access to these cameras.

But nonetheless, the second compilation step with ` bjam variant=release link=static` should generate an importable binary. It might be `PyHSCam.dll` (I don't remember exactly), and needs to be in the python import path along with the runtime requirements from the SDK.

**Comment by Julesghes** [comment]
Maybe the problem is trivial, I've never used Visual Studio before.

In the second part of the compilation, can you give me more details? Should I just create a folder with PyHSCam.cpp and jamroot.jam then in cmd run "bjam va... static" (if it's that bjam is not reconized by cmd)
or in VS create a project  


PS: in C:\boost\boost_1_61_0 I run "bjam va... static" and The Boost C++ Libraries were successfully built!

**Comment by ion201** [comment]
You should clone the repo and run bjam from the cloned directory. The bjam command is provided by boost, so that should be added to your path.

This project only requires the MSVC++ build tools, not the associated visual studio GUI/IDE

**Comment by Julesghes** [comment]
Hi,

I added bjam in the path and restart , and it works
I succeeded to use the camera, so i share some information.

to compile i use VS2015 x86 Native Tools Command Prompt from Visual Studio 2022
I also download BuildTools_Full.exe 
 
After i followed the instruction, created the folder , also added old .dll files from PFV3 (https://photron-fastcam-viewer.software.informer.com/3.5/), then only deleted the line 1476 (toolset.flags msvc.link.dll LINKFLAGS <suppress-import-lib>true : /NOENTRY ;), and run bjam.
and PyHSCam.pyd is created.

To run exemple.py, i have to authorized cmd.exe for the firewall and put the correct IP in control pannel ... 
And the camera photron SA3 is recording. 

Thank you for your help and your work
